extra-admin/xfsprogs/autobuild/build

```diff
@@ -2,6 +2,9 @@ abinfo 'Storing $SRCDIR value and unsetting $SRCDIR ...'
 export ABSRCDIR="$SRCDIR"
 unset SRCDIR
 
+abinfo "Replacing config.{guess,sub} ..."
+cp -v /usr/share/automake-1.16/config.{guess,sub} .
```

Comment:
Absolute path here. @liushuyu You might want to expose config.* replacement as a function as well.